napari/layers/_shapes_layer/shapes/path.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from .shape import Shape
+from ..shape_util import create_box
+
+
+class Path(Shape):
```

Comment:
Path? Arc? Spline? Line? :D

Reply:
Have improved to indicate sequence of line segments